Makefile

```diff
@@ -1,47 +1,63 @@
-ifndef GOPATH
-$(error No GOPATH set)
-endif
+SHORT_NAME := helm
```

Comment:
This variable is only used in one place.  Get rid of it and paste it's value in the one place it is used.